README.md

```diff
@@ -53,7 +53,63 @@ If you are a bit lost, the following terminology might help to clarify some aspe
 
 **Api** : A wrapper for the database — it allows a user of the emoji bar to store the click count ... somewhere, and then retrieve that amount when desired
 
+# Testing Methodology
+
+## Client
+
+No testing yet...
+
+## API
+
+No testing yet...
+
+## Snippet
+
+Currently only the **snippet** portion has automated testing setup.
+
+Now, the snippet is _not_ a JavaScript module, so the testing offered a challenge. The way it was solved is this case was to have a conditional export of each component that requires testing. You'll notice at the end of the snippet file `snippet/index.js` a series of lines that look like this:
+
+```javascript
+exports.pipe = pipe
+```
+
+This will make `pipe()` function importable from the `.test.js` files. It is set up as a conditional for when node is available.
+
+We will be using the standard file structure for jest which includes naming test files like: `filename.test.js`
+
+These files will be placed in the `snippet/__tests__` directory.
+
+Because of the small size of tested file we will be keeping test suites to single functions.
+
+### Testing Stack
+
+1. [Jest](http://facebook.github.io/jest/): is our testing framework. It has some issues, but it's the new guy in town and relatively easy to pick.
```

Comment:
Let's be kind to Jest, we don't need to say that it has some issues / is the new guy / easy to pick, a simple link is fine 😄